lib/rubyhub/configuration.rb

```diff
@@ -6,17 +6,19 @@
 module Rubyhub
   class Configuration
     include Singleton
-    attr_reader :options
+    attr_reader :options, :main_body
 
     CONFIG_PATH = '.rubyhub.yml'.freeze
+    DESCRIPTION_CONFIG_PATH = '.description.txt'.freeze
 
     def initialize
-      @options = self.class.exists? ? load_from_file : {}
+      @options = self.class.exists?(CONFIG_PATH) ? load_from_file : {}
+      @main_body = self.class.exists?(DESCRIPTION_CONFIG_PATH) ? read_description_from_file : {}
     end
 
     class << self
-      def exists?
-        File.exist?(CONFIG_PATH)
+      def exists?(file)
+        File.exist?(file)
       end
```

Comment:
This method is a bit strange. We just delegated `File.exists?` to `OurClass.exists?` and nothing more. I think it is not good approach, because now we need to all it `self.class.exists?` instead if simple `File.exists?`.

This method was about semantic `configuration exists?`. Now it does nothing useful.
Actually, I don't think we need to change it. It was checking for existence of configuration, right how this class called.